code/modules/reagents/chemistry/recipes/food.dm

```diff
@@ -102,6 +102,22 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/reagent_containers/food/snacks/dough(location)
 
+///Cookies by Ume
+
+/datum/chemical_reaction/cookiedough
+	name = "Dough"
+	id = "dough"
+	result = null
+	required_reagents = list("milk" = 10, "flour" = 10, "sugar" = 5)
```

Comment:
You're right, I tried to overlook if there were more recipes that used all that, but I missed that one. Good thing is I was planning on adding 5u of egg to the reaction, and I think only one or two recipes uses it as a reagent and not an item. I'll fix that.

Reply:
"Bake 10u Flour, 1 Egg, 5u Sugar, 5u Milk, 1 Golden Apple" the quantities are below the pastry dough reaction, and the apple tart as a baked recipe, none of them interfere with the other. I was thinking about adding the 5u egg, but if I want to make a chocolate flavored pastry dough, it would interfere with the chocolate pudding recipe. I'll think about it later on.